### PR TITLE
Edits to the Home Page

### PR DESCRIPTION
**Pull request by cjwerth**
Please review my edits to the home page template when you have a chance. Thanks!

### CODE REVIEW

File: front-page.php [hunk 1]
```diff
@@ -83,9 +83,8 @@
         <footer class="blockquote-footer">Don Baker, <cite title="The Washington Post Magazine">The Washington Post Magazine</cite></footer>
         </blockquote>
         <p>Farmville, Virginia’s former Robert Russa Moton High School, now a National Historic Landmark and museum, is the birthplace of America’s student-led civil rights revolution.</p>
-        <a href="" class="btn btn-primary">Mission</a>
-        <a href="" class="btn btn-primary">Board</a>
-        <a href="" class="btn btn-primary">Council</a>
+        <a href="/mission-vision" class="btn btn-primary">Mission & Vision</a>
```
Comment (JEverhart383) [inline front-page.php]: This url is actually /about/mission-vision/

File: front-page.php [hunk 2]
```diff
@@ -83,9 +83,8 @@
...
+        <a href="/board-council" class="btn btn-primary">Board & Council</a>
```
Comment (JEverhart383) [inline front-page.php]: href should be /about/board-council/

**Comment by cjwerth** [comment]
@JEverhart383 Fixed!

**Comment by JEverhart383** [comment]
@cjwerth That looks good I'm going to merge this and delete the branch. If you need to change anything else, please create a feature branch named like so: feature/change-footer-link

That way the name of the branch gives me a clue to what it contains. Thanks, JE

**Pull request merged**